src/queries.js

```diff
@@ -356,8 +387,12 @@ export {
   getAllByTitle,
   queryByValue,
   queryAllByValue,
+  queryByCurrentValue,
+  queryAllByCurrentValue,
```

Comment:
Could you move both of these down two lines. I kinda like to group things together.